backend/coins/eth/etherscan/etherscan.go

```diff
@@ -555,3 +555,8 @@ func (etherScan *EtherScan) SuggestGasPrice(ctx context.Context) (*big.Int, erro
 	}
 	return (*big.Int)(&result), nil
 }
+
+// SuggestGasTipCap implements rpc.Interface.
+func (etherScan *EtherScan) SuggestGasTipCap(ctx context.Context) (*big.Int, error) {
+	panic("not implemented")
```

Comment:
please let's not panic: `return nil, errors.New("not implemented")`

Reply:
done